File: help/eng_gb/exercicios_da_ficha_3/Ficha03_012/Ficha03_012.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha03_012 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Nome : " );
        String nome1 = keyb.nextLine();
        System.out.print("Nome : " );
        String nome2 = keyb.nextLine();
        System.out.print("Nomes Ordenados" );
        System.out.print("\n" + Math.min( nome1 , nome2 ) );
        System.out.print("\n" + Math.max( nome1 , nome2 ) );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Tue, 2022 Sep 13 21:57

File: help/eng_gb/exercicios_da_ficha_3/Ficha03_012/Ficha03_012.js
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("Nome : " );// print text
    nome1 = prompt("Nome : " ,"");
    
    document.write(nome1 + "\n");// print input 
    document.write("Nome : " );// print text
    nome2 = prompt("Nome : " ,"");
    
    document.write(nome2 + "\n");// print input 
    document.write("Nomes Ordenados" );
    document.write("\n" + Math.min( nome1 , nome2 ) );
    document.write("\n" + Math.max( nome1 , nome2 ) );
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Tue, 2022 Sep 13 21:57
